uportal-war/src/main/webapp/media/skins/respondr/common/less/header.less

```diff
@@ -24,6 +24,12 @@
     a {
         color: @header-link-color;
     }
+    #region-pre-header {
+       width:100%;
+       position: relative;
+       min-height:45px;
```

Comment:
Seems too thick to me, on viewing (though certainly a reasonable implementation choice).